File: src/main/java/no/nav/aareg/teknisk/historikk/consumer/aareg/dto/arbeidsforhold/Sporingsinformasjon.java
package no.nav.aareg.teknisk.historikk.consumer.aareg.dto.arbeidsforhold;

import java.time.LocalDateTime;

public record Sporingsinformasjon(LocalDateTime opprettetTidspunkt, LocalDateTime endretTidspunkt, String opprettetKilde, String endretKilde) {
}

File: src/main/kotlin/no/nav/aareg/teknisk_historikk/aareg_services/contract/Sporingsinformasjon.kt
package no.nav.aareg.teknisk_historikk.aareg_services.contract

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.time.LocalDateTime

@JsonIgnoreProperties(ignoreUnknown = true)
data class Sporingsinformasjon(
    val opprettetTidspunkt: LocalDateTime,
    val endretTidspunkt: LocalDateTime,
    val opprettetKilde: String,
    val endretKilde: String,
)